example/lib/main.dart

```diff
@@ -71,4 +71,9 @@ void main(List<String> arguments) {
   two - 1 == one;
   final another = one + 2;
   print('$another == $three');
+
+  final usLocalNumber = PhoneNumber.parse("(707) 555-1854", callerCountry: IsoCode.US);
```

Comment:
please add a test in the test file with this

Reply:
Done.